index.ts

```diff
@@ -13,160 +41,480 @@ enum Tile {
   KEY2, LOCK2
 }
 
-enum Input {
+interface Tile {
+  isAir(): boolean;
+  isFlux(): boolean;
+  isUnbreakable(): boolean;
+  isPlayer(): boolean;
+  isStone(): boolean;
+  isFallingStone(): boolean;
+  isBox(): boolean;
+  isFallingBox(): boolean;
+  isKey1(): boolean;
+  isLock1(): boolean;
+  isKey2(): boolean;
+  isLock2(): boolean;
+  draw(g: CanvasRenderingContext2D, x: number, y: number): void;
+  isEdible(): boolean;
+  isPushable(): boolean;
+  isStony(): boolean;
+  isBoxy(): boolean;
+  moveHorizontal(dx: number): void;
+  moveVertical(dx: number): void;
```

Comment:
```suggestion
  moveVertical(dy: number): void;
```
moveVertical의 인자값은 `dx`가 아닌 `dy`로 표기해야 합니다.

Reply:
dy 로 로컬 코드 수정했습니다. 지적 고맙습니다.
이를 언급하는 커밋은 따로 만들지 않겠습니다.